src/app/contexts/EventContext/useHandlerMap/index.js

```diff
@@ -0,0 +1,16 @@
+export const useHandlerMap = _map => e => {
```

Comment:
It's hard to really know what this function does and though the tests make sure it works as expected it doesn't give much indication to a real use case. At the very least maybe it's worth adding a short comment description about this function.